package.json

```diff
@@ -19,5 +18,8 @@
   ],
   "scripts": {
     "test": "make"
+  },
+  "dependencies": {
+    "chai": "^4.1.2"
```

Comment:
Updating the version looks good. This should still be listed as a `devDependency`, though.